syntax = "proto3";

package jhrpc;

message RpcHeader
{
    bytes service_name =1 ;
    bytes method_name =2 ;
    uint32 args_len =3;
}